src/Core/Mesh/ExtractIsoSurface/CMakeLists.txt

```diff
@@ -13,16 +13,31 @@ install(TARGETS ${PROJECT_NAME}
   COMPONENT Runtime
   )
 
-install(FILES Code.cxx CMakeLists.txt
+install(FILES Code.cxx CMakeLists.txt Code.py
   DESTINATION share/ITKSphinxExamples/Code/Core/Mesh/ExtractIsoSurface
   COMPONENT Code
   )
 
+set(input_volume ${CMAKE_CURRENT_BINARY_DIR}/HeadMRVolume.mha)
+set(output_mesh ${CMAKE_CURRENT_BINARY_DIR}/Output.vtk)
+set(lower_threshold 64)
+set(upper_threshold 255)
```

Comment:
Nitpick, could these please each be followed by an `unset` after their use to avoid accidentally propagating to other examples?

Reply:
I understand the intention to use unset to avoid interference with other examples, however, this content would be a major distraction in the CMakeLists.txt for those trying to learn from the individual example -- it is not related to the individual sample. This is much more important. Also, are you sure that this is actually going to cause issues with other examples due to CMake scoping via `add_subdirectory`?

Reply:
Ah, you are correct, scoping means this is not an issue. Good to leave as-is.